README.md

```diff
@@ -41,7 +41,7 @@ When pulling code changes from git, be sure to update your local packages from r
 
 ## Configuration Settings
 
-Default settings are loaded from `defaults.config`. To override these settings, create a configuration file and save the path to this file in the environment variable `PRINTAPP_SETTINGS`.
+Default settings are loaded from `src/printapp/config.py`. To override these settings, create a configuration file and save the path to this file (relative to the `src/printapp` directory) in the environment variable `PRINTAPP_SETTINGS`.
```

Comment:
Relative paths are a bad idea here. It's actually relative to where you launch the app from, so if you were to run the app from a different directory, relative paths won't work. I recommend using paths relative to home (`~/calvinwebprint/secret-config.py`) or absolute paths (`/home/calvinwebprint/secret-settings.py`). Perhaps replace the note about 'relative to' with an example. Perhaps use this line:

    (ex: `~/calvinwebprint/secret-settings.py`)